_RFCs/0001-rfc-for-rfcs.md

```diff
@@ -0,0 +1,185 @@
+
+  - Fork the [Core Plans Repository](https://github.com/habitat-sh/core-plans)
+  - Copy `_RFCs/0000-template.md` to `_RFCs/0000-my-feature.md` (where "my-feature" is descriptive. don't assign an RFC number yet).
+  - Fill in the RFC. Put care into the details: RFCs that do not present convincing motivation, demonstrate understanding of the impact of the design, or are disingenuous about the drawbacks or alternatives tend to be poorly-received.
+  - Submit a pull request ([make sure you sign your commits](https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md#signing-your-commits)). As a pull request the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.
+  - Each pull request will be labeled with the RFC tag, which will lead to its being triaged by the core-plans maintainers in a future meeting and assigned to one of it's members.
+  - Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments. Feel free to reach out to the RFC assignee directly to get help identifying stakeholders and obstacles.
+  - The maintainers will discuss the RFC pull request, as much as possible in the comment thread of the pull request itself. Offline discussion will be summarized on the pull request comment thread.
+  - RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, do not squash or rebase commits after they are visible on the pull request.
+  - At some point, a maintainer will propose a "motion for final comment period" (FCP), along with a *disposition* for the RFC (merge, close, or postpone).
+  - This step is taken when enough of the tradeoffs have been discussed that the subteam is in a position to make a decision. That does not require consensus amongst all participants in the RFC thread (which is usually impossible). However, the argument supporting the disposition on the RFC needs to have already been clearly articulated, and there should not be a strong consensus *against* that position outside of the maintainers. We will use our best judgment in taking this step, and the FCP itself ensures there is ample time and notification for stakeholders to push back if it is made prematurely.
+  - For RFCs with lengthy discussion, the motion to FCP is usually preceded by a *summary comment* trying to lay out the current state of the discussion and major tradeoffs/points of disagreement.
+  - Before actually entering FCP, *all* Core-Plans Maintainers must sign off; this is often the point at which many maintainers first review the RFC in full depth.
```

Comment:
I realize we have a process for maintainers becoming inactive in the event there is a non-responsive maintainer.  However, we should probably acknowledge that require all maintainers to sign off as oppose to a majority (2/3 or something), will slow down the process.  Not really suggesting we change that, but with a FCP of a fortnight it certainly implies we expect average submission to decision periods to take more than a couple weeks.

Reply:
That's true. Even in the case of this RFC we can see that out of 14 maintainers we've only seen responses from 6. I think perhaps switching this over to a "majority" vote is probably very reasonable.

Reply:
I might have missed where this is called out, but is a completed github review our definition of 'signing off' for this process?

Reply:
For now I think so. Ideally I want to set up a bot that allows maintainers to simply :+1: or :-1: in a clean way on an RFC